review.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <link rel="stylesheet" href="style.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Product Landing Page</title>
```

Comment:
Why is this being repeated for the three .html files you have? Kindly check

Reply:
* Kindly state why these is not a good idea the receiver might not know why he/she is been asked for changes

Reply:
I think no changes are required since this is the common head syntax for the HTML file.